src/components/TopicsList.jsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from "react-router-dom";
+
+const TopicsList = ({ topicsList, currTopic, setCurrTopic }) => {
+
+    return (
+        <nav className="Topics__nav">
+            {topicsList?.map((topic) => {
+                return (
+                    <Link to={`/?topic=${topic.slug}`} className={currTopic === topic.slug ? "Nav__link__selected" : "Nav__link"} key={topic.slug} onClick={() => {setCurrTopic(topic.slug)}}>{topic.slug.toUpperCase()}</Link>
```

Comment:
You're trying to match the query in your api here, but it would be clearer for a frontend suer to see a path like `/articles/football` there is no real need for this to be displayed like a query on the frontend